tests/docker/conftest.py

```diff
@@ -5,15 +5,32 @@
 from pytest import fixture
 from testcontainers.compose import DockerCompose
 from qiskit_serverless import ServerlessClient, QiskitFunction
+from qiskit_serverless.core.clients.local_client import LocalClient
 
 resources_path = os.path.join(
     os.path.dirname(os.path.abspath(__file__)), "source_files"
 )
 
 
+@fixture(scope="module", params=["serverless", "local"])
+def any_client(request):
```

Comment:
Instead to use `any_client` what do you think about using `base_client`? Not a strong opinion here but in reality all our clients inheritance from `BaseClient` if I remember correctly so I think it has sense.